index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <div class="header">
+      <a class="mylogo">My Logo</a>
+      <div class='nav'>
+        <img class='line' src="data:," alt>
+        <img class='line' src="data:," alt>
+	<img class='line' src="data:," alt>
+      </div>
+    </div>
+     	<h1 class="myheader">I’m PRATAP Glad to see you!</h1>
+    	<section>
+    	<h4 class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+    	</section>
+    	<a class="connect">LET’S CONNECT</a>
+	<ul class="social-media">
+
+	<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt='' title=''></a></li>
+	<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt='' title=''></a></li>
+	<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
+	<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt='' title=''></a></li>
+	<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
+	  
+	</ul>
```

Comment:
- [ ] Please wrap anything after the header in a`section` tag as requested by the previous reviewer